cli/config.go

```diff
@@ -87,35 +87,48 @@ func (c *Config) InitializeApp() error {
 		c.buildSchedulerMiddlewares(c.sh)
 	}
 
+	var err error
 	for name, j := range c.ExecJobs {
 		defaults.SetDefaults(j)
 		j.Client = c.dockerHandler.GetInternalDockerClient()
 		j.Name = name
 		j.buildMiddlewares()
-		c.sh.AddJob(j)
+		err = c.sh.AddJob(j)
+		if err != nil {
+			c.logger.Debugf("Unable to add new exec job %s from config: %v", name, err)
```

Comment:
Hmmm... the error was supposed to be logged inside `AddJob` function. Let me try to reproduce this